luminoso_api/json_stream.py

```diff
@@ -254,6 +260,13 @@ def _read_csv(reader, header, encode_fn):
         yield row_dict
 
 
+def convert_date(date_string, date_format):
+    if date_format != 'epoch':
+        return datetime.datetime.strptime(date_string, date_format).timestamp()
```

Comment:
I was trying to keep with the original behavior of "fail hard if any date doesn't work", but your idea is better.